packages/typography/src/ChartLabel.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { baseTypographyStyles, defaultTextColor } from './styles';
+import { CommonTypographyProps, Mode } from './types';
+
+/**
+ * ChartLabel
+ */
```

Comment:
This TSDoc should be on the function definition instead. Could we also have some description of what this component is, and how it should be used?